quotemachine/index.html

```diff
@@ -9,14 +9,14 @@
     <link rel="stylesheet" href="style.css">
 </head>
 
-<body class="random-color">
+<body class="random-color" style="transition:background-color 2s;">
     <div id="qbox">
-        <p id="quote-text" class="random-color">What programmer didn’t live with a deadline over his head?</p>
+        <p id="quote-text" class="random-text-color" style="transition:color 2s;"> What programmer didn’t live with a deadline over his head?</p>
 
-        <div id="buttons">
-            <img src="./twitter.svg" alt="twitter logo" class="icon random-color">
-           <img src="./tumblr.svg" alt="tumblr logo" class="icon random-color">
-            <p id="new-quote" class="new-quote random-color">New quote</p>
+        <div id="buttons" >
+            <img src="./twitter.svg" alt="twitter logo" class="icon random-color" style="transition:background-color 2s;">
+            <img src="./tumblr.svg" alt="tumblr logo" class="icon random-color" style="transition:background-color 2s;">
+            <p id="new-quote" class="new-quote random-color" style="transition:background-color 2s;">New quote</p>
```

Comment:
It's a button, do not use a paragraph element for something that you expect a user to click on